Source/Mosa.DeviceSystem/Image.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) MOSA Project. Licensed under the New BSD License.
+
+using Mosa.Runtime;
+
+namespace Mosa.DeviceSystem
+{
+	public class Image
+	{
+		public Pointer RawData;
+
+		public int Bpp;
+		public int Width;
+		public int Height;
+
+		public uint Size;
+
+		public Image(int width, int height, int depth)
+		{
+			Width = width;
+			Height = height;
+			Bpp = depth / 8;
+
+			Size = (uint)(width * height * Bpp);
+			RawData = GC.AllocateObject(Size);
```

Comment:
Hmmm. Need a better way. This will break GC since it’s not a true object. Just a note for now.